docs/topics/proc_deploying-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,30 @@
+[id='deploying-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+You can deploy a {VertX} application to Red Hat Enterprise Linux.
```

Comment:
If this procedure is supposed to be universal, it should say `{runtime}` instead of `{VertX}`. If it's for Vert.x only, the file name, anchor, and name should reflect that.